dsb-integration/src/test/java/com/spectralogic/dsbrowser/integration/LoggingServiceFake.java

```diff
@@ -0,0 +1,17 @@
+package com.spectralogic.dsbrowser.integration;
+
+import com.spectralogic.dsbrowser.api.services.logging.LogType;
+import com.spectralogic.dsbrowser.api.services.logging.LoggingService;
+import io.reactivex.Observable;
+
+public class LoggingServiceFake implements LoggingService {
```

Comment:
This looks like a duplicate of the one located in dsb-gui/src/test/java/com/spectralogic/dsbrowser/gui/LoggingServiceFake.java

Reply:
Shoot you're right, that isn't quite what I meant